stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/DefaultChatEventHandler.kt

```diff
@@ -90,4 +91,16 @@ public class DefaultChatEventHandler(private val channels: StateFlow<List<Channe
         event: NotificationRemovedFromChannelEvent,
         filter: FilterObject,
     ): EventHandlingResult = removeIfChannelIsPresent(channels, event.channel)
+
+    private fun handleChannelUpdate(channel: Channel): EventHandlingResult {
```

Comment:
I think we shouldn't modify this behavior in our default event handler. Our sample app use case is quite specific.